airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -183,15 +183,28 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
 
   @Override
   public JsonNode toDatabaseConfig(JsonNode config) {
+    List<String> additionalParameters = new ArrayList<>();
+
     final StringBuilder jdbc_url = new StringBuilder(String.format("jdbc:mysql://%s:%s/%s",
```

Comment:
Nitpick. Should use camel case: `jdbc_url` -> `jdbcUrl`.

Reply:
Updated, thanks!